### ISSUE TITLE
[BUG] Failed to parse Json error message

### ISSUE THREAD

**Issue by raphg51** [certtools/intelmq-manager#66]
Hello,

I ve already work with Intelmq in the past and i ll want to see whats news on intelmq but my installation don't want to work properly.

I've tried the installation on debian 8 abd ubuntu 14.04 and both installations fails the same way.

During installation I've no error but during the GUI utilization, I've trouble like this error message :

    Failed to obtain JSON: http://localhost/php/load_configs.php?file=defaults with error: parsererror, SyntaxError: JSON.parse: unexpected end of data at line 1 column 1 of the JSON data

Or this one :

    Error loading botnet status: SyntaxError: JSON.parse: unexpected end of data at line 1 column 1 of the JSON data

If i try to use the CLI of intelmq I've an other error message :+1: 
```bash
intelmqctl --h
Traceback (most recent call last):
  File "/usr/local/bin/intelmqctl", line 9, in <module>
    load_entry_point('intelmq==1.0.0.dev4', 'console_scripts', 'intelmqctl')()
  File "/usr/local/lib/python2.7/dist-packages/intelmq/bin/intelmqctl.py", line 531, in main
    x = IntelMQContoller()
  File "/usr/local/lib/python2.7/dist-packages/intelmq/bin/intelmqctl.py", line 157, in __init__
    logger = utils.log('intelmqctl', log_level='DEBUG')
  File "/usr/local/lib/python2.7/dist-packages/intelmq/lib/utils.py", line 240, in log
    handler = logging.FileHandler("%s/%s.log" % (log_path, name))
  File "/usr/lib/python2.7/logging/__init__.py", line 903, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 928, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: u'/opt/intelmq/var/log/intelmqctl.log'
```
So it seems that the installation is completely broken or i ve made a terrible mistake :).

My installation : 
```bash
apt-get install python python-pip
apt-get install git build-essential libcurl4-gnutls-dev libffi-dev libgnutls28-dev
apt-get install python-dev python-pycurl python-openssl python-pyasn1
apt-get install redis-server
sudo -s

git clone https://github.com/certtools/intelmq.git /tmp/intelmq
cd /tmp/intelmq

pip2 install -r REQUIREMENTS2
pip2 install .

useradd -d /opt/intelmq -U -s /bin/bash intelmq
echo 'export PATH="$PATH:$HOME/bin"' >> /opt/intelmq/.profile
chmod -R 0770 /opt/intelmq
chown -R intelmq.intelmq /opt/intelmq
```

Thx for your help.

**Comment by sebix** [comment]
Create the directory `/opt/intelmq/var/log/` as user intelmq. This is a bug in current master, I fixed that already in [ac00ec4c0a159e077395e8ea3debb26b0e98f1b6](https://github.com/sebix/intelmq/commit/ac00ec4c0a159e077395e8ea3debb26b0e98f1b6)
Note that master is a development version.

Please report errors on intelmq itself at https://github.com/certtools/intelmq/issues

**Comment by raphg51** [comment]
Finded!

Path /opt/intelmq/etc doesnt contain conf files (defaults.conf, pipeline.conf...) i ve copied it from /tmp to /opt and then all works properly. I ve i miss something during the installation?

**Comment by sebix** [comment]
`opt/intelmq/etc/example` should contain example configuration files.

**Comment by swannysec** [comment]
Can definitely confirm that the conf files were not correctly created.  Nor was /opt/intelmq/var/lib/bots/file-output/ .

**Comment by sebix** [comment]
Should be fixed with certtools/intelmq#465.
